argus/config/arestor.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Cloudbase Solutions Srl
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Config options available for the Arestor metadata service."""
+
+from oslo_config import cfg
+
+from argus.config import base as config_base
+
+
+class ArestorOptions(config_base.Options):
+
+    """Config options available for the Arestor metadata service."""
+
+    def __init__(self, config):
+        super(ArestorOptions, self).__init__(config,
+                                             group="arestor")
+        self._options = [
+            cfg.StrOpt(
+                "base_url",
+                default="http://0.0.0.0:8080",
```

Comment:
default should be localhost: 127.0.0.1 -- you cannot make a request to 0/0, you can only listen at 0/0